src/components/Logo/Logo.css

```
@@ -0,0 +1,14 @@
+.logo svg {
+	height: 8rem;
+	width: 8rem;
+}
+
+.sr-only:not(:focus):not(:active) {
+	clip: rect(0 0 0 0);
+	clip-path: inset(50%);
+	height: 1px;
+	overflow: hidden;
+	position: absolute;
+	white-space: nowrap;
+	width: 1px;
```

```suggestion
	clip: rect(0 0 0 0);
	clip-path: inset(50%);
	height: 1px;
	overflow: hidden;
	position: absolute;
	white-space: nowrap;
	width: 1px;
	color: transparent;
	pointer-events: none;
	user-select: none;
```

Those css properties actually disable any accidental clicks and text selection, as well as make the text transparent so there is no annoying pixel that may show up 😅